style/style.css

```diff
@@ -0,0 +1,518 @@
+* {
+  font-family: "Delius Unicase", cursive;
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  background: #ffffff;
+  user-select: none;
+}
+
+.container {
+  max-width: 1170px;
+  margin: 120px auto 120px auto;
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: fit-content fit-content fit-content;
+  gap: 120px 0px;
+  grid-template-areas:
+    "cities"
+    "nature"
+    "people";
+}
+
+div[class^="image"] {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover, cover;
+  border-radius: 20px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  padding: 20px;
+  grid-template-areas:
+    "icon-left icon-right"
+    "text-left text-right";
+}
+
+div[class^="image"]:hover {
+  filter: drop-shadow(0px 0px 30px rgba(0, 0, 0, 0.5));
+}
+
+div[class^="image"]:hover div[class^="icon"] {
+  transition: all 0.8s ease-out;
+  opacity: 1;
+}
+
+h1 {
+  font-family: "Delius Unicase";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 36px;
+  line-height: 45px;
+}
+
+div[class$="text-left"] {
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 23px;
+  color: #ffffff;
+}
+
+div[class$="text-right"] {
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 23px;
+  color: #e0e0e0;
+}
+
+/* ⇓ ⇓ ⇓ ⇓ ⇓ common icons and text labels for all images ⇓ ⇓ ⇓ ⇓ ⇓ */
+.icon-left {
+  grid-area: icon-left;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 20px, 20px;
+  background-image: url("../img/heart.svg");
+  background-color: #e0e0e0;
+  border-radius: 8px;
+  width: 40px;
+  height: 40px;
+  align-self: start;
+  justify-self: start;
+  cursor: pointer;
```

Comment:
You did use cursor pointer, so you understand those are buttons!)

Reply:
absolutely )